### DISCUSSION TITLE
How to remove nextjs version information

### DISCUSSION THREAD

**Discussion by DavisChang** [vercel/next.js#39488] [Help]
How to not display Nextjs version in Wappalyzer? or can add a config to remove Nextjs version. Thanks!

<img width="847" alt="截圖 2022-08-11 上午10 59 43" src="https://user-images.githubusercontent.com/15887183/184057950-26377cad-3f77-4cdd-8966-6cbd8275581e.png">

**Comment by conioX** [comment]
Hi guys, any update for this?

**Comment by phunguyen-ct** [comment]
Hi guys, any update on this? This has been marked as Vulnerable JavaScript Dependency from my Security team! I don't know how to solve this :(

**Reply by dominikager** [reply]
@phunguyen-ct i have the same problem.. any idea?

**Comment by jordibaliellas** [comment]
any updates about this?? 😢

**Comment by bstrzelczykLM** [comment]
This is still a potential security issue for apps using next.js (attackers see which next version is used and potentially can find exploits for this particular version). Is anyone in the next team looking into it?

**Comment by tungduonghgg123** [comment]
On Linux:
`grep -rl '14.2.3' .next/static/chunks | xargs sed -i  's/14.2.3/xx.xx.xx/g' `
On MacOS:
`grep -rl '14.2.3' .next/static/chunks | xargs sed -i '' 's/14.2.3/xx.xx.xx/g' `

Just replace 14.2.3 with your nextjs version. It will be replaced by xx.xx.xx
We run this bash command after we build nextjs in the docker. Seems working.
